src/datapump/sync/sync.py

```diff
@@ -325,6 +326,114 @@ def _should_update(self, latest_versions: Dict[str, str]) -> bool:
 
 
+class RADDAlertsSync(Sync):
+    """
+    Defines jobs to create new RADD alerts assets once a new release is available.
+    """
+
+    DATASET_NAME = "wur_radd_alerts"
+    SOURCE_BUCKET = "gfw_gee_export"
+    SOURCE_PREFIX = "wur_radd_alerts/"
+    INPUT_CALC = "(A >= 20000) * (A < 40000) * A"
+    NUMBER_OF_TILES = 115
+
+    def __init__(self, sync_version: str):
+        self.sync_version = sync_version
+
+    def build_jobs(self, config: DatapumpConfig) -> List[Job]:
+        """
+        Creates the WUR RADD raster layer and assets
+        """
+
+        latest_api_version = self._get_latest_api_version()
+
+        latest_release = self._get_latest_release()
+
+        if float(latest_api_version.lstrip("v")) >= float(latest_release.lstrip("v")):
+            return []
+
+        latest_release_dt: str = str(self.parse_version_as_dt(latest_release).date())
+
+        source_uris = [
+            f"gs://{self.SOURCE_BUCKET}/{self.SOURCE_PREFIX}{latest_release}"
+        ]
+
+        job = RasterVersionUpdateJob(
+            id=str(uuid1()),
+            status=JobStatus.starting,
+            dataset=self.DATASET_NAME,
+            version=latest_release,
+            tile_set_parameters=RasterTileSetParameters(
+                source_uri=source_uris,
+                calc=self.INPUT_CALC,
+                grid="10/100000",
+                data_type="uint16",
+                no_data=0,
+                pixel_meaning="date_conf",
+                band_count=1,
+                compute_stats=False,
+            ),
+            tile_cache_parameters=RasterTileCacheParameters(
+                max_zoom=14,
+                resampling="nearest",
+                symbology={"type": "date_conf_intensity"},
+            ),
+            content_date_range=ContentDateRange(
+                min="2014-12-31", max=str(latest_release_dt)
+            ),
+        )
+
+        return [job]
+
+    def _get_latest_api_version(self) -> str:
+        """
+        Get the version of the latest release in the Data API
+        """
+        client = DataApiClient()
+        return client.get_latest_version(self.DATASET_NAME)
+
+    def _get_latest_release(self) -> str:
+        """
+        Get the version of the latest *complete* release in GCS
+        """
+        versions: List[str] = get_gs_subfolders(self.SOURCE_BUCKET, self.SOURCE_PREFIX)
+
+        # Shouldn't need to look back through many, so avoid the corner
+        # case that would check every previous version when run right after
+        # increasing NUMBER_OF_TILES and hitting GCS as a new release is being
+        # uploaded
+        for version in sorted(versions)[:3]:
+            version_prefix = "/".join((self.SOURCE_PREFIX, version))
+            version_tiles: int = len(
+                get_gs_files(self.SOURCE_BUCKET, version_prefix, extensions=[".tif"])
+            )
+            if version_tiles > self.NUMBER_OF_TILES:
+                raise Exception(
```

Comment:
Mmm, I wasn't sure what effect what effect it might have, TBH. I think in this particular case we would want to get attention so we know something changed or broke.